### PR TITLE
feat(toolbar): Add new visibility-related options.

### PR DESCRIPTION
**Pull request by guusdk**
Two changes that allow the visibility of the toolbar to be configured.

Here _might_ be dragons: Please note that this is the very first time I've made unsupervised changes to a React Native project. Please review accordingly.

I've tested these changes on two different Android devices, but not on IOS. Strictly speaking, I did not test by changing the content of  `interface_config.js`. Instead, I copied the existing code that reads from there, and switched the boolean values in the first few lines of the `_getInitialState` method in `react/features/toolbox/reducer.js` to simulate changing the configuration.

### CODE REVIEW

**Comment by virtuacoplenny** [comment]
Drive by comment: there is actually a config option called "alwaysVisibleToolbar" that is used by the integration tests. Its value may get passed into setToolboxAlwaysVisible and maybe can get you what you want for the always-visible config option. Whether you should be using it or I don't know, but it exists.

**Comment by saghul** [comment]
Jenkins, please test.

**Comment by saghul** [comment]
Jenkins please test this.

**Comment by guusdk** [comment]
(Try "pretty please")

File: react/features/conference/components/Conference.native.js [hunk 1]
```diff
@@ -378,13 +389,15 @@ function _mapDispatchToProps(dispatch) {
...
- *     _toolboxVisible: boolean
+ *     _toolboxVisible: boolean,
+ *     _toolboxAlwaysVisible: boolean
  * }}
  */
 function _mapStateToProps(state) {
     const { connecting, connection } = state['features/base/connection'];
     const { conference, joining, leaving } = state['features/base/conference'];
     const { reducedUI } = state['features/base/responsive-ui'];
+    const { visible, alwaysVisible } = state['features/toolbox'];
```
Comment (saghul) [inline react/features/conference/components/Conference.native.js]: style: can you please list the variables in alphabetical order?

**Comment by guusdk** [comment]
Test results seem to be somewhat unstable. Since the last successful test run, I've only re-ordered the definition of two variables. It's extremely unlikely that this causes a test to fail.

**Comment by guusdk** [comment]
Having played with it a little, making the desktop and mobile clients behave the same in this respect does not necessarily make much sense. Is there precedent in the UI config for web or mobile-specific settings?

**Comment by saghul** [comment]
After reading on this, I'd rather only take the `alwaysVisible` part of this PR. Both web and mobile show the toolbar initially already, and more code would be needed in order to prevent this (from a cursory look). Is that OK with you?

> Having played with it a little, making the desktop and mobile clients behave the same in this respect does not necessarily make much sense. Is there precedent in the UI config for web or mobile-specific settings?

Not that I can think of.

**Comment by guusdk** [comment]
> Is that OK with you?

Yup. Do you want me to rework the PR?

**Comment by saghul** [comment]
Yeah, please.

**Comment by guusdk** [comment]
I've now removed the "show toolbar initially" part of the PR (keeping only the "alwaysVisible" bit) and rebased to the current master.

**Comment by guusdk** [comment]
I've rebased to the current master just now.

**Comment by saghul** [comment]
Jenkins please test this please.

**Comment by havid0707** [comment]
@saghul @guusdk this case replace ```config.js->alwaysVisibleToolbar``` with ```interface_config.js->TOOLBAR_ALWAYS_VISIBLE```.

I integrate with the JItsi Meet through the API, and overwrite the ```interfaceConfigOverwrite.TOOLBAR_ALWAYS_VISIBLE = true```,  but the toolbar will fade out after timeoutMS.

Before this，```configOverwrite.alwaysVisibleToolbar``` works well for me.
Whether this changeset has been considered ```through the API```？

**Comment by saghul** [comment]
The API uses the internal machinery, so no special changes should be required. Please open a new issue if this problem persists.

**Pull request merged**